components/centraldashboard/app/api_test.ts

```diff
@@ -371,7 +379,7 @@ describe('Dashboard API', () => {
         [header]: `${prefix}test@testdomain.com`,
       };
       const response = await sendTestRequest(url, headers, 405, 'post');
-      expect(response).toEqual({error: 'Unexpected error creating profile'});
+      expect(JSON.stringify(response)).toEqual(JSON.stringify({error: 'Unexpected error creating profile'}));
```

Comment:
The stringify calls are unnecessary as `.toEqual` does a deep equality test.

Reply:
Fixed, that was from a test I was doing to see why a test was failing.